.github/workflows/ci.yml

```diff
@@ -28,6 +29,13 @@ jobs:
       -
         name: Run chart-testing (lint)
         run: ct lint --all --validate-maintainers=false charts/
+      - 
+        name: Run helm charts unittests
```

Comment:
I think Helm is already available on the runner, see also [this](https://github.com/rancher/fleet/pull/3457/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f) which only installs the `helm-unittest` plugin.

Reply:
Oh, great! I didn’t see that PR was also introducing unit tests for the Helm charts, thanks!

I’ve updated it to match that PR, with the exception of the `-f` flag. (I mentioned in the PR that if the files have the `_test.yaml` suffix, the plugin loads them automatically.)

Reply:
Nice, thanks :)
It seems there's a couple of superfluous env vars now; once removed, this should be ready to merge.